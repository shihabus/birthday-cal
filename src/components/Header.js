import React from "react";
import styled from "styled-components";

const HeaderText = styled.div`
  grid-area: header;
  display: flex;
  justify-content: center;
  align-items: flex-end;
  font-size: 2.5rem;
  color: #677580;
`;

export default function Header(props) {
  return <HeaderText>{props.header}</HeaderText>;
}
