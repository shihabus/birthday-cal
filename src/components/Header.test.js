import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import Header from "./Header";

test("Header snapshot", () => {
  const { container } = render(<Header />);
  expect(container.firstChild).toMatchSnapshot();
});
