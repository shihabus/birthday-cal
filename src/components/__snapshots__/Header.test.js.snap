// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<div
  class="sc-AxjAm heRJOY"
/>
`;
